Release: v1.27.2 (#12249)
filecoin-project/lotus: CHANGELOG.md

## Improvements

# v1.27.2 / 2024-07-17

> This is a release candidate of Lotus v1.27.2. This will be an OPTIONAL Lotus release. It contains some improvements that are relevant for node operators that are using or serving `eth_*` RPC methods. It also contains an upgraded libp2p to v0.35.3 which is included in this release for additional testing of some fixes that may solve some connectivity problems experienced by some users (See [libp2p/go-libp2p#2858](https://github.com/libp2p/go-libp2p/issues/2858) for more information).

Reviewer: ```suggestion
This is the stable release of Lotus v1.27.2. This will be an OPTIONAL Lotus release. It contains some improvements that are relevant for node operators that are using or serving `eth_*` RPC methods. It also contains an upgraded libp2p to v0.35.3 which is included in this release for additional testing of some fixes that may solve some connectivity problems experienced by some users (See [libp2p/go-libp2p#2858](https://github.com/libp2p/go-libp2p/issues/2858) for more information).
```
